acceptancetests/jujupy/k8s_provider/base.py

```diff
@@ -114,7 +116,7 @@ def _tear_down_substrate(self):
         """
         raise NotImplementedError()
 
-    def __init__(self, bs_manager, cluster_name=None, timeout=1800):
+    def __init__(self, bs_manager, cluster_name=None, enable_rbac=False, timeout=1800):
```

Comment:
Maybe this should be true by default

Reply:
I guess that means we'd need to make the jenkins job do "--disable-rbac" instead of "--enable-rbac"

Was thinking it's better to be more restrictive by default, but it probably doesn't matter

Reply:
Given that kubeflow still does not enable RBAC yet, we should probably disable it for now unless we know we want to enable it.